src/js/media-fullscreen-button.js

```diff
@@ -15,31 +15,28 @@ const exitFullscreenIcon = `
 </svg>
 `;
 
+const api = {
+  enter: "requestFullscreen",
+  exit: "exitFullscreen",
+  event: "fullscreenchange",
+  element: "fullscreenElement",
+  error: "fullscreenerror",
+};
+
+if (!document.fullscreenElement) {
+  api.enter = "webkitRequestFullScreen";
+  api.exit = document.webkitExitFullscreen != null ? "webkitExitFullscreen" : "webkitCancelFullScreen";
+  api.event = "webkitfullscreenchange";
+  api.element = "webkitFullscreenElement";
+  api.error = "webkitfullscreenerror";
+}
+
```

Comment:
- Moved the api mapping out of the function as it only needs to be run once.
- Removed `MS` and `moz` cases as they don't apply to the list of minimum supported browser versions.

Reply:
Nice, thanks for the cleanup pass.